### PR TITLE
feat(CI): add main.yml github actions workflow

### PR DESCRIPTION
**Pull request by Trdls**

### CODE REVIEW

**Comment by Trdls** [comment]
Tests won't pass until we add env variables UPCLOUD_API_TEST_USER & UPCLOUD_API_TEST_PASSWORD need to create a test account and add those to repo secrets

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  pull_request:
+    branches:
+      - master
+jobs:
+  Build:
+    strategy:
+      matrix:
+        os-php-versions: [ 
+          { os: ubuntu-16.04, php: 5.6 },
+          { os: ubuntu-16.04, php: 7.0 },
```
Comment (tohalla) [inline .github/workflows/main.yml:15]: point to latest as with others?

**Review by tohalla — APPROVED** [review]
Can be merged after updating the badge and the travis config has been removed

File: .github/workflows/main.yml (hunk 1 above)
Comment (Trdls) [reply .github/workflows/main.yml:15]: Latest is from 7.1 upwards https://github.com/marketplace/actions/setup-php-action

**Pull request merged**